content/release-notes/2023-05-11-console.md

```diff
@@ -0,0 +1,15 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Adjusted how Autoscaling minimum and maximum compute sizes are displayed in the **Compute endpoints** table on the branch details page.
+- UI: Added a **History retention** metric to the **Storage** widget on the Neon **Dashboard**. By default, Neon retains a 7-day history of data changes to support [point-in-time restore](../docs/reference/glossary#point-in-time-restore). History retention will become a configurable setting in a future release.
+- UI: Enhanced the **Request Downgrade** modal, which is accessible from the **Billing** page in the console.
```

Comment:
Should we mention that it is now automated and the operation happens instantly?

Reply:
Good idea. Please take a look at the revised note.